extractor/src/test/java/org/schabi/newpipe/extractor/services/youtube/YoutubeKioskExtractorTest.java

```diff
@@ -3,22 +3,305 @@
 import static org.schabi.newpipe.extractor.ServiceList.YouTube;
 import static org.schabi.newpipe.extractor.services.DefaultTests.assertNoMoreItems;
+import static org.schabi.newpipe.extractor.services.DefaultTests.defaultTestMoreItems;
 import static org.schabi.newpipe.extractor.services.DefaultTests.defaultTestRelatedItems;
 
 import org.junit.jupiter.api.Test;
 import org.schabi.newpipe.extractor.exceptions.ParsingException;
 import org.schabi.newpipe.extractor.services.BaseListExtractorTest;
 import org.schabi.newpipe.extractor.services.DefaultSimpleExtractorTest;
-import org.schabi.newpipe.extractor.services.youtube.extractors.YoutubeTrendingExtractor;
+import org.schabi.newpipe.extractor.services.youtube.extractors.kiosk.YoutubeLiveExtractor;
+import org.schabi.newpipe.extractor.services.youtube.extractors.kiosk.YoutubeTrendingExtractor;
+import org.schabi.newpipe.extractor.services.youtube.extractors.kiosk.YoutubeTrendingGamingVideosExtractor;
+import org.schabi.newpipe.extractor.services.youtube.extractors.kiosk.YoutubeTrendingMoviesAndShowsTrailersExtractor;
+import org.schabi.newpipe.extractor.services.youtube.extractors.kiosk.YoutubeTrendingMusicExtractor;
+import org.schabi.newpipe.extractor.services.youtube.extractors.kiosk.YoutubeTrendingPodcastsEpisodesExtractor;
 
 public class YoutubeKioskExtractorTest {
 
+    public static class Live extends DefaultSimpleExtractorTest<YoutubeLiveExtractor>
+            implements BaseListExtractorTest, InitYoutubeTest {
+        @Override
+        protected YoutubeLiveExtractor createExtractor() throws Exception {
+            return (YoutubeLiveExtractor) YouTube.getKioskList().getDefaultKioskExtractor();
+        }
+
+        @Override
+        @Test
+        public void testServiceId() {
+            assertEquals(YouTube.getServiceId(), extractor().getServiceId());
+        }
+
+        @Override
+        @Test
+        public void testName() throws Exception {
+            assertEquals("Live", extractor().getName());
+        }
+
+        @Override
+        @Test
+        public void testId() throws Exception {
+            assertEquals("live", extractor().getId());
+        }
+
+        @Override
+        @Test
+        public void testUrl() throws Exception {
+            assertEquals(
+                    "https://www.youtube.com/channel/UC4R8DWoMoI7CAwX8_LjQHig/livetab?ss=CKEK",
+                    extractor().getUrl());
+        }
+
+        @Override
+        @Test
+        public void testOriginalUrl() throws Exception {
+            assertEquals(
+                    "https://www.youtube.com/channel/UC4R8DWoMoI7CAwX8_LjQHig/livetab?ss=CKEK",
+                    extractor().getOriginalUrl());
+        }
+
+        @Override
+        @Test
+        public void testRelatedItems() throws Exception {
+            // As there is sometimes very recently ended livestreams present, we can't test whether
+            // all streams are running live streams
```

Comment:
We could test that there is at least 10% of livestreams? But yeah this is just a possible improvement for later